docs/user/aws/install_upi.md

```diff
@@ -0,0 +1,276 @@
+# Install: User Provided Infrastructure (UPI)
+
+The steps for performing a UPI-based install are outlined here. Several [CloudFormation][cloudformation] templates are
+provided to assist in completing these steps or to help model your own.
+
+## Create Ignition Configs
+
+The machines will be started manually. Therefore, it is required to generate the bootstrap and machine Ignition configs
+and store them for later steps.
+
+```console
+$ openshift-install-linux-amd64 create ignition-configs
+? SSH Public Key /home/user_id/.ssh/id_rsa.pub
+? Platform aws
+? Region us-east-1
+? Base Domain example.com
+? Cluster Name openshift
+? Pull Secret [? for help]
+```
+
+After running the command, several files will be available in the directory.
+
+```console
+$ tree
+.
+├── auth
+│   └── kubeconfig
+├── bootstrap.ign
+├── master.ign
+├── metadata.json
+└── worker.ign
+```
+
+## Create/Identify the VPC to be Used
+
+You may create a VPC with various desirable characteristics for your situation (VPN, route tables, etc.). The
+VPC configuration and a CloudFormation template is provided [here](../../../upi/aws/cloudformation/01_vpc.yaml).
+
+A created VPC via the template or manually should approximate a setup similar to this:
+
+![Example VPC Architecture Diagram](images/install_upi_vpc.png)
+
+## Create DNS entries and Load Balancers for Control Plane Components
+
+The DNS and load balancer configuration within a CloudFormation template is provided
+[here](../../../upi/aws/cloudformation/02_cluster_infra.yaml). It uses a public hosted zone and creates a private hosted
+zone similar to the IPI installation method. It also creates load balancers and listeners the same way as the IPI
+installation method. This template can be run multiple times within a single VPC and in combination with the VPC
+template provided above.
+
+### Optional: Manually Create Load Balancer Configuration
+
+It is needed to create a TCP load balancer for ports 6443 and 22623. The targets will be the master nodes.
+Port 6443 must be accessible to both clients external to the cluster and nodes within the cluster. Port 22623 must
```

Comment:
same comment i had [here](https://github.com/openshift/installer/pull/1416#pullrequestreview-215161275) w.r.t port 22623 which is not K8s _standard_

maybe something like `Port 22623 (used by ignition-config) must ....`

Reply:
> maybe something like Port 22623 (used by ignition-config) must ....

I added descriptions for both ports with dafdaa0 -> bdae4f9.

Reply:
@wking  many thanks ! 🌟 💯